targets/TARGET_STM/mbed_overrides.c

```diff
@@ -129,11 +129,14 @@ void mbed_sdk_init()
         LL_HSEM_1StepLock(HSEM, CFG_HW_STOP_MODE_SEMID);
         /*Release HSEM in order to notify the CPU2(CM4)*/
         LL_HSEM_ReleaseLock(HSEM, CFG_HW_STOP_MODE_SEMID, 0);
-    } else {
+    }
+#ifndef NO_FORCED_CM4_BOOT
+    else {
```

Comment:
I feel like it is not needed:
Code is under DUAL_CORE switch. So CM4 need to boot somehow.
* Either it boots at reset in parallel of CM7(depending on Option bytes),
and in this case CM4 automatically go to sleep mode, waiting for CM7 to configure clock (because it has been decided in Mbed that only CM7 is able to set clock), Then CM7 need to wakeup CM4. This correspond to `if (READ_BIT(SYSCFG->UR1, SYSCFG_UR1_BCM4)`
* Or CM4 doesn't boot at reset (depending on Option bytes) and CM7 need to force CM4 boot. This is the `esle`

Reply:
I don't understand why booting the CM4 should be compulsory :slightly_smiling_face: 
In fact, we are leaving the chance to boot it later when RPC lib requires it or to flash a CM7 only binary that acts as an H743 (more or less)